baiji/config.py

```diff
@@ -76,6 +94,13 @@ def key(self):
     def secret(self):
         return self._try(['AWS_SECRET_ACCESS_KEY', 'AWS_SECRET'], 'AWS_SECRET')
 
+    @property
+    def region(self):
+        return self._try(['AWS_DEFAULT_REGION'], 'REGION')
+
+# Alias to avoid breaking changes
+Credentials = Settings
```

Comment:
Yea, it could be a minor. The only other way I would think to structure it is that the `load` function for `Settings` calls into the `load` function for `Credentials` and extends the result.